api/apps/api/src/modules/scenario-specification/features-found.event.ts

```diff
@@ -0,0 +1,20 @@
+import { IEvent } from '@nestjs/cqrs';
+import { SpecificationOperation } from './specification-operation';
+
+/**
+ * Once features were assigned/created one shall update the Specification
+ */
+export class FeaturesFound implements IEvent {
+  constructor(
+    public readonly specificationId: string,
+    public readonly spec: {
```

Comment:
```suggestion
    public readonly specification: {
```
and I'd extract the specification as a separate type

Reply:
fixed in https://github.com/Vizzuality/marxan-cloud/pull/371/commits/4dbad00ade491ccca947032a295834751eab9b2d